deployment.rb

```diff
@@ -91,6 +91,7 @@ def load_configuration()
     'dynamo_tables_table'         => "#{rack_env}_tables",
     'dynamo_tables_index'         => "channel_id-table_name-index",
     'dynamo_properties_table'     => "#{rack_env}_properties",
+    'throttle_data_apis'          => rack_env == :production,
```

Comment:
According to dashboard-console, `CDO.geocoder_redis_url` is defined in production but not on staging/test. To enable throttling on staging/test I think I'd first want to understand how redis is accessed there. Is it simply running locally on `localhost:6379` ?